portal/app/api/v1/endpoint/farm_device.py

```diff
@@ -20,4 +26,12 @@ def get_queryset(self, **kwargs):
         return queryset
 
 
+class FarmDeviceDataViewSet(viewsets.ModelViewSet):
+    authentication_classes = (CsrfExemptAuth.CsrfExemptSessionAuthentication, TokenAuthentication)
+    serializer_class = FarmDeviceDataSerializer
+
+    def get_queryset(self, **kwargs):
+        queryset = FarmData.objects.filter(home__owner__user=self.request.user).order_by('id')
```

Comment:
I'd expect you would want the latest (most recent) data first instead of oldest. Meaning, ordering by descending instead of ascending.
So it would be `order_by('-id')` vs `order_by('id')`